python/py-cucumber-tag-expressions/Portfile

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-tag-expressions-python
+version             2.0.2
+revision            0
+categories-append   devel
+
+license             MIT
+maintainers         nomaintainer
+platforms           darwin
+
+description         Tag Expression Parser for Python
+
+long_description    Cucumber can be used to implement automated tests\
+                    based on scenarios described in your Gherkin\
+                    feature files.
+
+homepage            https://github.com/cucumber/tag-expressions-python
+
+master_sites        pypi:p/${python.rootname}/
+distname            ${python.rootname}-${version}
+
+checksums           rmd160  77f0b50aff47cab4994deb94d8f3c9ac00d05a67 \
+                    sha256  35fb5e9445a2913b7a02a7ab691dd0acdf3d5664118717d2b9c1becec9497eea \
+                    size    32498
+
+supported_arch      noarch
```

Comment:
this is a typo and should be ```supported_archs```, this breaks the CI. Again, and I repeat myself here, don't push code without checking it locally

Reply:
Thanks done.